requirements.txt

```diff
@@ -24,6 +24,7 @@ passlib>=1.6.2,<1.7
 lockfile>=0.10.2,<0.11
 python-gnupg>=0.3.7,<0.4
 jsonpath-rw>=1.3.0
+ldap
```

Comment:
Hmm ... looks like there is no package ldap on pypi. Do you want to use https://pypi.python.org/pypi/python-ldap?

This is also most likely the root cause of the travis and scrutinizer failures.